continuousflex/protocols/protocol_pdb_dimred.py

```diff
@@ -74,111 +61,101 @@ def _defineParams(self, form):
-            NPE <k=12>
-               Neighborhood Preserving Embedding, k=number of nearest neighbours 
-        """)
-        form.addParam('extraParams', params.StringParam, default=None,
-                      expertLevel=params.LEVEL_ADVANCED,
-                      label='Extra params',
-                      help='These parameters are there to change the default parameters of a dimensionality reduction'
-                           ' method. Check xmipp_matrix_dimred for full details.')
-
+        form.addParam('setOfPDBs', params.PointerParam, pointerClass='SetOfPDBs, SetOfAtomStructs',
+                      condition='pdbSource == 2',
+                      label="Set of PDBs",
+                      help='Use a scipion object SetOfPDBs / SetOfAtomStructs')
+        form.addParam('dcds_file', params.PathParam,
+                      condition='pdbSource == 3',
+                      label="List of trajectory DCD files",
+                      help='Use the file pattern as file location with /*.dcd')
+        form.addParam('dcd_start', params.IntParam, default=0,
+                      condition='pdbSource == 3',
+                      label="Beginning of the trajectory",
+                      help='Index of the desired begining of the trajectory')
+        form.addParam('dcd_end', params.IntParam, default=-1,
+                      condition='pdbSource == 3',
+                      label="Ending of the trajectory",
+                      help='Index of the desired end of the trajectory')
+        form.addParam('dcd_step', params.IntParam, default=1,
+                      condition='pdbSource == 3',
+                      label="Step of the trajectory",
+                      help='Step to skip points in the trajectory')
+        form.addParam('dcd_ref_pdb', params.PointerParam, pointerClass='AtomStruct',
+                      condition='pdbSource == 3',
+                      label="trajectory Reference PDB",
+                      help='Reference PDB of the trajectory')
+
+        form.addSection(label='Principal Component Analysis')
         form.addParam('reducedDim', IntParam, default=2,
-                      label='Reduced dimension')
-
-        # form.addParallelSection(threads=0, mpi=8)
+                      label='Number of Principal Components')
+        form.addParam('alignPDBs', params.BooleanParam, default=False,
+                      label="Align PDBs ?",
+                      help='Perform rigid body alignement on the set of PDBs to a reference PDB')
+        form.addParam('alignRefPDB', params.PointerParam, pointerClass='AtomStruct',
+                      condition='alignPDBs',
+                      label="Alignement Reference PDB",
+                      help='Reference PDB to align the PDBs with')
+        form.addParam('matchingType', params.EnumParam, label="Match structures ?", default=0,
+                      choices=['Both structures are the same', 'Match chain name/residue num/atom name',
```

Comment:
"Both" -> "All" 
"same" -> ?
For same, we need to find a smart word. Maybe something like "have 1-1 correspondence between the atoms in the same order"?

Reply:
What about _"All structures are matching"_, and in the help I say _"Method to find atomic coordinates correspondence between the trajectory coordinates and the reference PDB. The method will select the matching atoms and sort them in the corresponding order. If the structures in the files are already matching, choose "All structures are matching"_